diff --git a/wp-admin/network/site-new.php b/wp-admin/network/site-new.php

--- a/wp-admin/network/site-new.php
+++ b/wp-admin/network/site-new.php
@@ -40,7 +40,7 @@ if ( isset($_REQUEST['action']) && 'add-site' == $_REQUEST['action'] ) {
 		wp_die( __( 'Can&#8217;t create an empty site.' ) );
 	$blog = $_POST['blog'];
 	$domain = '';
-	if ( preg_match( '|^([a-zA-Z0-9-])+$|', $blog['domain'] ) )
+	if ( preg_match( '|^([a-zA-Z0-9-/])+$|', $blog['domain'] ) )
 		$domain = strtolower( $blog['domain'] );
 
 	// If not a subdomain install, make sure the domain isn't a reserved word
